srcpkgs/singularity/template

```diff
@@ -0,0 +1,54 @@
+# Template file for 'singularity'
+pkgname=singularity
+version=3.11.0
+revision=1
+build_style=go
+go_import_path=github.com/sylabs.io/singularity
+go_mod_mode=vendor
```

Comment:
why set the build_style when you override all the steps?

Reply:
I was trying to do minimal change to https://github.com/void-linux/void-packages/pull/28129, and I agree there is no need to set it.